litrpc/proxy.proto

```diff
@@ -54,4 +54,7 @@ message GetInfoRequest {
 message GetInfoResponse {
     // The version of the LiTd software that the node is running.
     string version = 1;
+
+    // The git commit hash of the litd binary.
+    string commit_hash = 2;
```

Comment:
Let me know if you think `commit` would be a more suitable name for this, as the contents may be the `tag` rather than a commit hash. But opted for this naming as it then mimics the `loop` getinfo response :)